hernia--steps-to-surgery/cover.html

```diff
@@ -19,17 +19,44 @@
 
       <!-- booking CTA -->
       <a href="https://www.ebs.ncrs.nhs.uk/"
-         style="top: 687px;
+         style="top: 798px;
+                left: 105px;
+                width: 402px;
+                height: 64px;"></a>
+
+      <!-- what to ask your surgeon -->
+
+          <a name="what-to-ask"
+         style="top: 900px;
+                left: 105px;
+                width: 250px;
+                height: 30px;"></a>
+
+
+      <a href="see-the-surgeon.html"
+         style="top: 1158px;
+                left: 105px;
+                width: 250px;
+                height: 30px;"></a>
+
+      <!-- Checking you are fit for surgey --> checking-you-are-fit
```

Comment:
this text is outside the comment and currently shows up at the bottom of the document